config/routes.rb

```diff
@@ -19,6 +19,10 @@
     resource :user, only: [], path: '/' do
       post :update_myself, on: :member
     end
+
+    resource :user, only: [], path: '/' do
+      post :update_org, on: :member
```

Comment:
Since this resource block is duplicated with L19-21, this block should be removed and you can move this line to just after L20.